docs/contributing/Pull_Requests.mdx

```diff
@@ -74,4 +74,7 @@ We generally review PRs oldest to newest, unless we consider a newer PR higher p
 
 Once we have reviewed your PR, we will provide any feedback that needs addressing.
 If you feel a requested change is wrong, don't be afraid to discuss with us in the comments.
-Once the feedback is addressed, and the PR is reviewed, we'll ensure the branch is up to date with `main`, and merge it for you.
+
+Once you've addressed all our feedback by making code changes and/or started a followup discussion, [re-request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews#re-requesting-a-review) from each maintainer whose feedback you addressed.
```

Comment:
@mahdi-farnia are there any other points in the process you'd want explicit docs on? Including things you figured out but had to infer from prior knowledge / looking at other PRs / etc.?

Reply:
@JoshuaKGoldberg adding my two cents here, perhaps you could also you clarify in the docs when and who should push the `Resolve conversation` button in a PR. Is it the reviewer or the original PR owner?

Reply:
Ahhhhh yes thank you! I keep forgetting that little tidbit.